draft-ietf-rats-corim.md

```diff
@@ -957,24 +957,30 @@ If a `flags-map` value is not specified, then the operational mode is unknown.
 
 The following describes each member of the `flags-map`:
 
-* `configured` (index 0): The measured environment is fully configured for
-  normal operation if the flag is true.
+* `configured` (index 0): If the flag is true, the measured environment is fully configured for normal operation.
 
-* `secure` (index 1): The measured environment's configurable security settings
-  are fully enabled if the flag is true.
+* `secure` (index 1): If the flag is true, the measured environment's configurable
+security settings are fully enabled.
 
-* `recovery` (index 2): The measured environment is NOT in a recovery state if
-  the flag is true.
+* `recovery` (index 2): If the flag is true, the measured environment is NOT in recovery
+mode.
 
-* `debug` (index 3): The measured environment is in a debug enabled state if
-  the flag is true.
+* `debug` (index 3): If the flag is true, the measured environment is in a debug enabled
+mode.
 
-* `replay-protected` (index 4): The measured environment is protected from
-  replay by a previous image that differs from the current image if the flag is
-  true.
+* `replay-protected` (index 4): If the flag is true, the measured environment is
+protected from replay by a previous image that differs from the current image.
 
-* `integrity-protected` (index 5): The measured environment is protected from
-  unauthorized update if the flag is true.
+* `integrity-protected` (index 5): If the flag is true, the measured environment is
+protected from unauthorized update.
+
+* `runtime-meas` (index 6): If the flag is true, the measured environment is measured
+after being loaded into memory.
+
+* `immutable` (index 7): If the flag is true, the measured environment is immutable.
```

Comment:
As shown in issue #63, the names in TCG extension are `immutable-meas` and `tcb-meas`. Why use different names here? Or how about change them to be easier to understand, like `runtime-measured`, `is-immutable`, and `is-tcb`.

By the way, is `tcb` redundant? Since `replay-protected` and `integrity-protected` already implies that the attester is a TCB.

I think we need new flag `confidentiality-protected`.

Reply:
> I think we need new flag `confidentiality-protected`.

I think this should be a new issue.

Reply:
> Or how about change them to be easier to understand, like `runtime-measured`, `is-immutable`, and `is-tcb`.

:+1:

Reply:
fixed in latest push.